File: year2020/month11/day1102/ms_1102.cs
using System;

namespace algorithm_91
{
    class ms_1102
    {
        static void Main(string[] args)
        {
            String data = Console.ReadLine();

            if(data.Length <= 100) 
            {
                String[] input = data.Split(' ');
                int maxIndex = 0;
                
                for(int i = 0; i < input.Length; i++) 
                    if(input[i].Length >= input[maxIndex].Length)
                        maxIndex = i;

                Console.WriteLine(input[maxIndex].Length);
            }
        }
    }
}

File: year2020/month11/day1102/ms_1102.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main() {
	string input;

	getline(cin, input);

	if (input.length() <= 100) {
		stringstream ss(input);
		string data[100];
		int maxIndex = 0;
		int i = 0;

		while (getline(ss, input, ' '))
			data[i] = input;

		for (i = 0; i < input.length(); i++)
			if (data[i].length() >= data[maxIndex].length())
				maxIndex = i;

		cout << data[maxIndex].length() << endl;
	}
	return 0;
}